router/connection_maker.go

```diff
@@ -304,14 +304,16 @@ func (cm *ConnectionMaker) attemptConnection(address string, acceptNewPeer bool)
 }
 
 func tryImmediately() (time.Time, time.Duration) {
-	interval := time.Duration(rand.Int63n(int64(InitialInterval)))
-	return time.Now(), interval
+	return time.Now(), InitialInterval
 }
 
+// The delay at the nth retry is a random value in the range
+// [i-i/2,i+i/2], where i = InitialInterval * 1.5^(n-1).
```

Comment:
This comment is only true indirectly, because of the way the function is called.  Maybe `tryAfter` and `tryImmediately` should be methods on `Target*`, and manipulate `tryInterval` directly?